build/release/Jenkinsfile

```diff
@@ -41,6 +41,8 @@ pipeline {
                   sh "gh auth login --with-token < ${tokenFile}"
                   sh "gh pr create --title '${message}' --body '${message}' --head ${targetBranch} --base ${env.BRANCH_NAME}"
                 }
+                // publish site from 'mvn release' run
+                maven cmd: "site:stage scm-publish:publish-scm"
```

Comment:
no site generation (would be in wrong mode -> next snapshot).
I think the generated site from release-profile should still exist and be re-usable for publishing (but hard to test without actually running a release :)